references/kvm-admin-api/test/integration/features/support/init.js

```diff
@@ -21,8 +21,12 @@ before(function () {
   this.apickli = new apickli.Apickli(
     "https",
     process.env.APIGEE_ORG + "-" + process.env.APIGEE_ENV +
-    ".apigee.net/kvm-admin/v1"
+    ".apigee.net/kvm-admin/v1/organizations/" +
```

Comment:
The edge hostname will not work with the GCP token validation. We should use something like an `env.APIGEE_HOSTNAME` variable instead.

Reply:
Fixed